README.md

```diff
@@ -137,6 +137,16 @@ If you want to help with filters translations, you can join us on Crowdin: https
 
 Please learn more about translating our products: https://kb.adguard.com/general/adguard-translations
 
+## <a id="templates"></a> Filters templates
+`@include` directive allows to include the content of specified file into the filter.
+
+If you want to add a filter and keep all its rules during optimization, use the `/notOptimized` parameter.
+
+Example:
+```
+@include "https://example.org/test-filter.txt" /notOptimized
```

Comment:
Can we use this already?

Reply:
@Alex-302, we use this parameter for few years already.
for example: https://github.com/AdguardTeam/FiltersRegistry/blob/master/filters/filter_2_English/template.txt#L68